msal/src/test/java/com/microsoft/identity/nativeauth/NativeAuthPublicClientApplicationJavaTest.java

```diff
@@ -120,6 +138,30 @@ public class NativeAuthPublicClientApplicationJavaTest extends PublicClientAppli
     private final char[] password = "verySafePassword".toCharArray();
     private final String code = "1234";
     private final String emptyString = "";
+    private final boolean allowPII;
+
+    public NativeAuthPublicClientApplicationJavaTest(boolean allowPII) {
+        this.allowPII = allowPII;
+    }
+
+    private final List<String> sensitivePIIMessages = Arrays.asList(
```

Comment:
can we add one more item here? A regex for a JWT, which would cover access tokens and ID tokens. For that regex you can omit the `=` and `:` parts.

Reply:
Done. a JWT token or similar structure.`"""^[A-Za-z0-9_-]+\\.[A-Za-z0-9_-]+\\.[A-Za-z0-9_-]+$""" // JWT token`